src/app/body/task-details/task-details.component.css

```diff
@@ -18,9 +18,41 @@
     background-color: #5A20F0;
 }
 
+#sprint {
+    color: #5A20F0;
+}
+
 @media (max-width: 991px) {
     .navbar-nav {
         float: right;
         flex-direction: row;
     }
```

Comment:
this is forcing the screen to adjust for smaller screen, do we really need this.

Reply:
I didn't add these lines bro. Will ask for guidance and do the needful.